### PR TITLE
fix: feature digital credentials

### PR DESCRIPTION
**Pull request by amanji**
*Issue #:* /bcgov/entity#18521

*Description of changes:*

* fixes discrepancy between queue message types.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

**Review by severinbeauvais — COMMENTED** [review]
Looks fine to me. Does it need to be deployed to Test?

**Comment by amanji** [comment]
Yes please. Just need to confirm it works there

**Comment by severinbeauvais** [comment]
What about the lint and test errors?

**Comment by severinbeauvais** [comment]
Akiff, at some point, can you use a fresh branch, or hard-reset it to main, so that the list of commits above goes away? :)

**Comment by severinbeauvais** [comment]
> Yes please. Just need to confirm it works there

If Argus doesn't get to this by early afternoon, I will do it.

File: queue_services/entity-digital-credentials/src/entity_digital_credentials/worker.py [hunk 1]
```diff
@@ -69,8 +69,8 @@ async def process_digital_credential(dc_msg: dict, flask_app: Flask):
             f'bc.registry.business.{FilingCore.FilingTypes.CHANGEOFREGISTRATION.value}',
             f'bc.registry.business.{FilingCore.FilingTypes.DISSOLUTION.value}',
             f'bc.registry.business.{FilingCore.FilingTypes.PUTBACKON.value}',
-            'bc.registry.admin.bn',
-            'bc.registry.admin.manual'
+            'bc.registry.business.bn',
+            'bc.registry.business.manual'
```
Comment (argush3) [inline queue_services/entity-digital-credentials/src/entity_digital_credentials/worker.py:73]: I agree with `bc.registry.business.bn`.  

I think we should replace `bc.registry.admin.manual` with `bc.registry.admin.revoke`.  My thought is we should still distinguish admin based operations.  We'll need to update the cli to support this in some way.

**Comment by amanji** [comment]
Closing this as it has been addressed in #2330

**Pull request closed without merging**